src/lib/y2_alternatives/dialog/list_alternatives.rb

```diff
@@ -104,10 +104,14 @@ def update_table(index)
       end
 
       def accept_handler
-        @alternatives_list.each(&:save)
+        @alternatives_list.each(&:save) if permissions?
         finish_dialog
       end
 
+      def permissions?
```

Comment:
rails way is actually both, `save` and `save!` with first one returning false and second one raising exception.